xptifw/basic_test/CMakeLists.txt

```diff
@@ -6,11 +6,11 @@ add_executable(XPTIFWBasicTests ${SOURCES})
 target_link_libraries(XPTIFWBasicTests PRIVATE xptifw ${CMAKE_DL_LIBS})
 
 if (UNIX)
-  target_compile_definitions(XPTIFWBasicTests PRIVATE -gline-tables-only -O3)
+  add_compile_options(-gline-tables-only -O3)
```

Comment:
could you please clarify what this change stands for?

Reply:
That line used to cause CMake errors and has been changed to ensure we have -O3 for the basic performance tests